manifest.go

```diff
@@ -146,25 +158,66 @@ type fallbackManifestFileV1 struct {
 	AddedSnapshotID *int64 `avro:"added_snapshot_id"`
 }
 
-func (f *fallbackManifestFileV1) toManifest() *manifestFileV1 {
-	f.manifestFileV1.AddedSnapshotID = *f.AddedSnapshotID
+func (f *fallbackManifestFileV1) toFile() *manifestFile {
+	if f.AddedSnapshotID == nil {
+		f.manifestFileV1.AddedSnapshotID = -1
+	}
 
-	return &f.manifestFileV1
+	return f.manifestFileV1.toFile()
 }
 
 type manifestFileV1 struct {
-	Path               string          `avro:"manifest_path"`
-	Len                int64           `avro:"manifest_length"`
-	SpecID             int32           `avro:"partition_spec_id"`
-	AddedSnapshotID    int64           `avro:"added_snapshot_id"`
-	AddedFilesCount    *int32          `avro:"added_files_count"`
-	ExistingFilesCount *int32          `avro:"existing_files_count"`
-	DeletedFilesCount  *int32          `avro:"deleted_files_count"`
-	AddedRowsCount     *int64          `avro:"added_rows_count"`
-	ExistingRowsCount  *int64          `avro:"existing_rows_count"`
-	DeletedRowsCount   *int64          `avro:"deleted_rows_count"`
-	PartitionList      *[]FieldSummary `avro:"partitions"`
-	Key                []byte          `avro:"key_metadata"`
+	manifestFile
+	AddedFilesCount    *int32 `avro:"added_files_count"`
+	ExistingFilesCount *int32 `avro:"existing_files_count"`
+	DeletedFilesCount  *int32 `avro:"deleted_files_count"`
+	AddedRowsCount     *int64 `avro:"added_rows_count"`
+	ExistingRowsCount  *int64 `avro:"existing_rows_count"`
+	DeletedRowsCount   *int64 `avro:"deleted_rows_count"`
+}
+
+func (m *manifestFileV1) toFile() *manifestFile {
+	m.manifestFile.version = 1
+	m.Content = ManifestContentData
+	m.SeqNumber, m.MinSeqNumber = 0, 0
```

Comment:
should this be 0 or -1? 
0 should represent the INITIAL_SEQUENCE_NUMBER
-1 should represent unassigned

Reply:
In `NewManifestFile`, V1 table's seqNum is null

Reply:
according to the [iceberg spec](https://iceberg.apache.org/spec/#manifest-lists) we should use 0 when reading a V1 table's sequence number. So that's why I'm using it here.

Reply:
ok makes sense, this is same as pyiceberg's [INITIAL_SEQUENCE_NUMBER](https://grep.app/search?f.repo=apache%2Ficeberg-python&f.repo.pattern=ice&q=INITIAL_SEQUENCE_NUMBER)

Reply:
nit: let's use INITIAL_SEQUENCE_NUMBER or something similar to provide more context

Reply:
done